### PR TITLE
ABC-329 Fix for 2.3.X: Normals AlwaysCompute Option does not work

### PR DESCRIPTION
**Pull request by RoseHirigoyen**
JIRA TICKET: https://jira.unity3d.com/browse/ABC-334

Bug:
- The "AlwaysCompute Normals" was not correctly applied because the old enum popup assumed enums started at 0.

Solution:
- Use the unity enum popup (instead of a custom one) that uses the specified enum value

Tests:
 - Test that the AlwaysCompute option properly computes normals of a broken triangle, and test that the Compute if missing does not modify them

Doc:
- Added an entry in the changelog
- Does not modify the UI

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.formats.alembic?pullRequest=521) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.formats.alembic?pullRequest=521) before we can accept your contribution.<br/><hr/>**noreply@unity3d.com** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.formats.alembic?pullRequest=521) it.</sub>

**Review by cguer — APPROVED** [review]
QA approved 👍!

Tested on:
- 2023.1.0a21 (Windows only)

**Review by lucillecaillaud — APPROVED** [review]
Didn't test, but code looks good to me!!
If I get it, it was mostly a UI issue.

File: com.unity.formats.alembic/CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fix the Normals import settings of the Importer.
```
Comment (MaryPerchanok) [inline com.unity.formats.alembic/CHANGELOG.md:11]: Suggestion:
- Fixed the Normals import settings of the Importer.

**Review by MaryPerchanok — COMMENTED** [review]
For the Changelog, change Fix to Fixed.

**Review by MaryPerchanok — APPROVED** [review]

**Pull request merged**